contracts/src/internal/HyperdriveLP.sol

```diff
@@ -564,4 +567,40 @@ abstract contract HyperdriveLP is
             _marketState.bondReserves = updatedBondReserves.toUint128();
         }
     }
+
+    function _updateLiquiditySafe(int256 _shareReservesDelta) internal returns (bool) {
```

Comment:
Is there a preferred way to do this over simply re-implementing the logic w/ `calculateUpdateLiquiditySafe`?